src/python/turicreate/test/test_recommender.py

```diff
@@ -1353,6 +1353,14 @@ def test_implicit(self):
             self._test_coreml_export(m4, ['135085','135038'])
         '''
 
+    def test_recommend_from_interactions(self):
+        data=tc.SFrame({'userId':[1,1,1,2,2,2,3,3,3],'movieId':[10,11,12,10,13,14,10,11,14]})
+        exclude_pairs=tc.SFrame({'movieId':[14]})
+        observed_data=tc.SFrame({'movieId':[10]})
+        data_set=tc.item_similarity_recommender.create(data,user_id='userId',item_id='movieId')
```

Comment:
This is confusing. What you're creating on this line is a model not a data set. I would just call the variable `model` rather than `data_set`.